src/catalog/catalog.cpp

```diff
@@ -67,8 +68,8 @@ void Catalog::CreateMetricsCatalog() {
 void Catalog::CreateCatalogDatabase() {
   storage::Database *database = new storage::Database(START_OID);
```

Comment:
I don't think we should do this now because I think we are planning to do a catalog rewrite next semester anyway.